src/Nelmio/Alice/Persister/Doctrine.php

```diff
@@ -47,12 +61,6 @@ public function persist(array $objects)
      */
     public function find($class, $id)
     {
-        $entity = $this->om->find($class, $id);
-
-        if (!$entity) {
```

Comment:
I'm fine with returning null in general, but this *is* a breaking change. Anyone catching the exception would have their code break, so I don't think we can just do it "lightly".